### ISSUE TITLE
Error is thrown when using with skyrim AE v1.6.640

### ISSUE THREAD

**Issue by AbhiSharma1210** [max-su-2019/ModernStaggerLock#1]
I am using MO2 (latest version) on Skyrim SE v1.6.640. When trying to launch the game using SKSE loader,  the plugin throws an error (see screenshot). 

I have installed all dependencies and ran Nemesis (also selected Modern Stagger Lock in the mod list on Nemesis).
The game runs correctly when I disable MSL in MO2.
![ModernStaggerLockError](https://github.com/max-su-2019/ModernStaggerLock/assets/96657549/d575e1f3-4526-497b-a3d6-480fe595e186)

**Comment by zune127** [comment]
Getting this error too

**Comment by TheDreamer21160** [comment]
I have the same problem, but in addition, the documents related to the animations of the falmer/draugr creatures are not recognized by Nemesis.

**Comment by lDimondl** [comment]
> 

you have to get the https://www.nexusmods.com/skyrimspecialedition/mods/45966

**Comment by AbhiSharma1210** [comment]
> > 
> 
> you have to get the https://www.nexusmods.com/skyrimspecialedition/mods/45966

Already have it already and I already wrote that "I have all dependencies". But then again how does Nemesis Creature going to fix an outdated DLL. Please explain.

**Comment by TheDreamer21160** [comment]
> > > 
> > 
> > 
> > you have to get the https://www.nexusmods.com/skyrimspecialedition/mods/45966
> 
> Already have it already and I already wrote that "I have all dependencies". But then again how does Nemesis Creature going to fix an outdated DLL. Please explain.

Indeed, so I installed it and it solves the problems related to creatures, but the error message remains related to compatibility with 1.6.640

**Comment by Zack23121432** [comment]
Will this ever get fixed?

**Comment by diriLeSh** [comment]
Still not fixed i have both, but dll file needs update for 1.6.640

**Comment by VTJ-Dumay-Pothuon** [comment]
Same issue with GOG version. Yeah it's just the dll version check, I'm not even sure any update is required per se…

**Comment by LifelessLego** [comment]
Hello. I have a problem. Everything is in the photo.

![image](https://github.com/max-su-2019/ModernStaggerLock/assets/136930757/43686e86-3631-486e-8a47-c7aa9c1ae211)

**Comment by LifelessLego** [comment]
I have all files but I don't understand what I have to do to fix this.. :c

**Comment by LifelessLego** [comment]
Help me please.. I wish you a good day/night!

**Comment by LifelessLego** [comment]
Or I don't know where to find that file but please.. help me :c

**Comment by tyman66** [comment]
This has been an issue since July. You're telling me no one has updated the version check to include 1.6.640?

**Comment by ifeelagood** [comment]
Lemme try compile for 1.6.640

**Comment by ifeelagood** [comment]
Sadly stuck in dependency hell (vcpkg isnt used for commonlib) and cant figure it out. I might have another crack later

**Comment by lobotomy-x** [comment]
> Sadly stuck in dependency hell (vcpkg isnt used for commonlib) and cant figure it out. I might have another crack later

I have been trying in my spare time to setup commonlib so I can patch some existing mods but havent been able to build anything except some templates due to vcpkg/cmake issues. Can you elaborate on the issue @ifeelagood? Will gladly patch this if I get it setup

**Comment by colossalsthegame** [comment]
Same issue

**Comment by Me1onica** [comment]
I don't think the author of the mod will ever fix it, it's been a VERY long time.

**Comment by Shinjiri** [comment]
i even dont know how to install it xD
i thought via MO2, but if do so, the data is invalid and tryed with everything to turn it to data, but nothing.
do i have to install it manually ?

**Comment by yxrizzz** [comment]
sadly theres no way to fix it until maxsu drop us DLL files for 1.6.629+ version

**Comment by Ankit-Maj** [comment]
https://github.com/SkyHorizon3/ModernStaggerLock/releases/tag/v1.07 Found a version compatible with 1.6.629+
